test/data/matgas/A2.m

```diff
@@ -15,7 +15,6 @@
 mgc.is_per_unit                  = 0;
```

Comment:
I modified this file to improve formatting and numerics in the examples pull request.  I think you can safely discard these changes in lue of what was merged into master there